### PR TITLE
bump alpine version with iptalbes to 3.10.3

### PR DESCRIPTION
**Pull request by DockToFuture**
**What this PR does / why we need it**:
bump alpine version with iptalbes to 3.10.3

**Release note**:

```improvement operator
none
```

### CODE REVIEW

**Review by vpnachev — DISMISSED** [review]
Out of curiosity, how/where is this image built?
Yesterday when I created #1640 I wanted to update this one as well, but only `3.8` and `3.9` were available.

File: charts/images.yaml [hunk 1]
```diff
@@ -113,7 +113,7 @@ images:
   tag: "3.10.3"
 - name: alpine-iptables
   repository: eu.gcr.io/gardener-project/alpine-iptables
-  tag: "3.9"
+  tag: "3.10.3"
```
Comment (vpnachev) [inline charts/images.yaml:117]:
```suggestion
  sourceRepository: <...>
  tag: "3.10.3"
```

**Review by vpnachev — COMMENTED** [review]
It will be even better if you provide the `sourceRepository` like it is done for `nginx-ingress-controller`  for example

**Review by vpnachev — APPROVED** [review]

File: charts/images.yaml [hunk 2]
```diff
@@ -112,8 +112,9 @@ images:
   repository: alpine
   tag: "3.10.3"
 - name: alpine-iptables
+  sourceRepository: github.com/DockToFuture/alpine-iptables
```
Comment (ialidzhikov) [inline charts/images.yaml:115]: Shouldn't this image Dockerfile go in a repo under gardener org and to be released by ci/cd, not manually? For example gardener/vpn?

**Pull request merged**